data/src/main/java/com/nemethedgar/data/features/news/local/NewsEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.nemethedgar.data.features.news.local;
+
+import androidx.annotation.Nullable;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+@Entity(tableName = "news")
+public class NewsEntity {
+
+    @PrimaryKey(autoGenerate = true)
+    @Nullable
+    public Integer id;
+
+    public String imageUrl;
+
+    public String title;
+
+    public String content;
+
+    public String description;
```

Comment:
let's make all the members final, besides the id: immutable classes are less prone to bugs, considering their state does not change trough their lifecycle

Reply:
done